app/src/main/java/com/pddstudio/highlightjs/demo/SyntaxActivity.java

```diff
@@ -33,16 +34,16 @@ public class SyntaxActivity extends AppCompatActivity implements
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_syntax);
-        FileObject fileObject = (FileObject) getIntent().getExtras().getSerializable("fileObject");
-        if(getActionBar() != null) {
-            assert fileObject != null;
-            getActionBar().setTitle(fileObject.getFileName());
+//        FileObject fileObject = (FileObject) getIntent().getExtras().getSerializable("fileObject");
```

Comment:
simplifying the sample

Reply:
IMO you can remove commented code if you don't think we will need it in the future.